### PR TITLE
AYS-376 | Update fields in emergency evacuation applications page

### PR DESCRIPTION
**Pull request by murat-es**
### Checklist

Before submitting your pull request, ensure the following:

- [ ] **Change Type:**
- [ ] Bug fix
- [x] Feature addition
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Other (please specify)

- [ ] **Summary:**

  - Add a short description (e.g., added or modified features).

- [x] **Title and Branch Naming Conventions:**

  - The pull request title follows the
    standard: [Pull Request Naming Conventions](CONTRIBUTING.md#pull-request-naming-conventions).
  - The branch name follows one of the conventions outlined in
    the [Branch Naming Conventions](CONTRIBUTING.md#branch-naming-conventions).

- [x] **Local Testing:**

  - I have tested my changes locally on my browser, and they are working as expected.

- [x] **Code Quality:**

  - The code is formatted according to the project's coding guidelines and style.
  - The code has been reviewed to ensure its quality.
  - The code does not contain any issues flagged by ESLint.

- [ ] **Documentation:**

  - Necessary documentation has been added or existing documentation has been updated.

- [x] **Reviewers and Assignees:**

  - Default reviewers have been assigned to this pull request.
  - Assignees have been added if necessary.

  - [ ] **Related Issue:**
  - Closes #AYS-345 (If there is a relevant issue)

- [x] **Labels and Associations:**
  - No specific actions are required in the Labels and Associations section for this pull request.

### CODE REVIEW

File: src/modules/emergencyEvacuationApplications/components/columns.tsx [hunk 1]
```diff
@@ -35,17 +38,28 @@ export const columns: (
...
     {
       accessorKey: 'phoneNumber',
       header: () => i18next.t('phoneNumber'),
-      cell: ({ row }) => formatPhoneNumber(row.original.phoneNumber),
+      cell: ({ row }) =>
+        formatPhoneNumber(
+          row.original.isInPerson
+            ? row.original.phoneNumber
+            : row.original.applicantPhoneNumber
+        ),
```
Comment (sanshigo345) [inline src/modules/emergencyEvacuationApplications/components/columns.tsx:62]: tıpatıp aynı logic 3 kere art arda yazmaktansa, genel bir fonksiyon yazılıp o fonksiyon kullanılabilir

File: src/modules/emergencyEvacuationApplications/components/columns.tsx [hunk 2]
```diff
@@ -13,6 +13,9 @@ import { getSortState } from '@/lib/getSortState'
 export interface EmergencyEvacuationApplication {
   id: string
   referenceNumber: string
+  applicantFirstName?: string
+  applicantLastName?: string
+  applicantPhoneNumber?: PhoneNumber
   firstName: string
   lastName: string
   phoneNumber: PhoneNumber
```
Comment (sanshigo345) [inline src/modules/emergencyEvacuationApplications/components/columns.tsx:21]: mantıken bu alanların da zorunlu olmaması gerekmez mi

**Review by sanshigo345 — CHANGES_REQUESTED** [review]

File: src/modules/emergencyEvacuationApplications/components/columns.tsx (hunk 2 above)
Comment (murat-es) [reply src/modules/emergencyEvacuationApplications/components/columns.tsx:21]: https://test-servis.afetyonetimsistemi.org/api/v1/emergency-evacuation-applications apisinden gelen veride bu alanlar her zaman dolu olarak geliyor. Benim interface'e eklediğim alanlar ise isInPerson alanı "true" olduğu zaman "null" olarak geliyor.

Comment (sanshigo345) [reply src/modules/emergencyEvacuationApplications/components/columns.tsx:21]: buradaki type definition ile api arasında bi bağlantı var mı?

**Review by sanshigo345 — APPROVED** [review]

**Pull request merged**